services/ory-svc/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"common"
+	"context"
+	"encoding/json"
+	"gen/proto/libs/events/v1"
+	pb "gen/proto/services/ory_svc/v1"
+	daprc "github.com/dapr/go-sdk/client"
+	daprcmn "github.com/dapr/go-sdk/service/common"
+	daprd "github.com/dapr/go-sdk/service/http"
+	"github.com/google/uuid"
+	zlog "github.com/rs/zerolog/log"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"hwutil"
+	"net/http"
+)
+
+const ServiceName = "ory-svc"
+
+// Version is set at compile time
+var Version string
+
+var DaprPubsub string
+
+var daprClient *daprc.GRPCClient
+
+func main() {
+	common.Setup(ServiceName, Version, false)
+	DaprPubsub = hwutil.GetEnvOr("DAPR_PUBSUB", "pubsub")
+
+	daprClient = common.MustNewDaprGRPCClient()
+
+	addr := hwutil.GetAddr()
+	service := daprd.NewService(addr)
+
+	if err := service.AddServiceInvocationHandler("/after_registration_webhook", afterRegistrationWebhookHandler); err != nil {
+		zlog.Fatal().Str("endpoint", "after_registration_webhook").Err(err).Msg("could not add service invocation handler")
+	}
+
+	if err := service.Start(); err != nil {
+		zlog.Fatal().Str("addr", addr).Err(err).Msg("could not start http server")
+	}
+}
+
+func afterRegistrationWebhookHandler(ctx context.Context, in *daprcmn.InvocationEvent) (out *daprcmn.Content, err error) {
+	if in.Verb != http.MethodPost {
+		return nil, status.Error(codes.InvalidArgument, "invalid method")
```

Comment:
if an err is returned, dapr will always assume something went wrong server-side and return a 500 [(relevant code)](https://github.com/dapr/go-sdk/blob/c5377994aa5f6ffd6d64fc3ee7d2bba7a9124018/service/http/invoke.go#L67-L69).

other than that, we return a status.Error here, which is from gRPC land, though not really relevant 
it is semantically wrong and may give the wrong impression when reading the code

last thing: because this is not part of the grpc chain I don't think it is using the logger middleware to log returned errors (though I might be wrong here)